src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonChoiceType.php

```diff
@@ -107,4 +100,25 @@ public function getBlockPrefix()
     {
         return 'sylius_taxon_choice';
     }
+
+    /**
+     * @param null|string   $root
+     * @param null|callable $filter
```

Comment:
`string|null` and `callable|null`. Also we should not align parameters vertically.